app/controllers/api/years_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  class YearsController < Api::BaseController
+
+    # GET /api/years.json
+    def index
+      plural_resource_name = "@years"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.